lib/resources/dstu2/medication_order.yaml

```diff
@@ -24,7 +23,6 @@ fields:
 
 - name: medicationCodeableConcept
   required: 'No'
```

Comment:
an old cardinality of 1..1 does not mesh with this field being un-required. can you double check our implementation?

Reply:
[double checked](https://github.com/cerner/fhir.cerner.com/pull/374/commits/6fb0b4a5a1bdcd0b07e086f542d44ded83b12430?file-filters%5B%5D=.yaml#diff-e19db23d6092046f31617ecce00ba335L25)